sktime/forecasting/model_selection/_tune.py

```diff
@@ -1900,7 +1900,15 @@ def get_test_params(cls, parameter_set="default"):
             "scoring": "MeanAbsolutePercentageError(symmetric=True)",
             "update_behaviour": "no_update",
         }
-        return [params, params2, params3]
+        scorer_with_lower_is_better_false = MeanAbsolutePercentageError(symmetric=True)
```

Comment:
This is fine for testing, but so counterintuitive!

A different example, say with a function that basically returns negative of smape will be preferable to me.

Reply:
I think there is no class that currently has `lower_is_better` as `False`.